2018/07.md

```diff
@@ -91,15 +91,16 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
     | stage | timebox | topic | presenter |
     |:-----:|:-------:|-------|-----------|
+    | 2     | 60m     | Decorators for Stage 3 | Daniel Ehrenberg |
 
-1. Longer or open-ended discussions
+1. Longer or open-ended discussions (Thursday afternoon)
```

Comment:
this isn't necessarily going to be the case; we might get to them sooner, or not at all.